web-components/src/components/modal/Modal.test.ts

```diff
@@ -216,6 +214,5 @@ describe("Modal Component", () => {
     jest.runAllTimers();
 
     expect(document.activeElement).not.toEqual(input);
```

Comment:
These tests might need rethinking. `document.activeElement()` cannot penetrate the shadowDom, and will always return the outermost custom element, and so this test assertion will always return true.  You'd either need to test in a different way, or replace the native `activeElement` with something that can recursively search into shadowDoms.

Reply:
Kevin, I agree. And when Kostay will available I'll ask him about logic, Or rewrite tests for modal component.